parity-clib/examples/cpp/main.cpp

```diff
@@ -129,7 +109,8 @@ int parity_subscribe_to_websocket(void* parity) {
 
 	std::vector<const void*> sessions;
 
-	Callback cb { .type = CALLBACK_WS, .counter = ws_subscriptions.size() };
+	Callback cb { .type = CALLBACK_WS };
+	callback_counter = ws_subscriptions.size();
 
 	for (auto sub : ws_subscriptions) {
 		void *const session = parity_subscribe_ws(parity, sub.c_str(), sub.length(), callback, &cb);
```

Comment:
Again, the use of `void *` is an anti-pattern.  This should be wrapped in a type-safe object that cleans up the resources in its destructor.